docs/user-guide/map-templates.md

```diff
@@ -0,0 +1,66 @@
+
+For each template in the *Map Templates* list the following buttons are displayed:
+
+<img src="../img/map-templates/map-templates-buttons.jpg" class="ms-docimage" style="max-width:500px;"/>
+
+* The **Replace** button <img src="../img/button/replace-button.jpg" class="ms-docbutton" /> allows the user to replace the current map context with the template, as follows:
+
+<img src="../img/map-templates/replace-map.gif" class="ms-docimage"/>
+
+* The **Add Template** button <img src="../img/button/add_to_map_button.jpg" class="ms-docbutton" /> allows the user to add the template to current map context, as follows:
+
+<img src="../img/map-templates/add-templates.gif" class="ms-docimage"/>
+
+* The **Add to favorites** button <img src="../img/button/favourites-button.jpg" class="ms-docbutton" /> allows the user to add the template to favorites on top of the list
+
+## Enabling the Map templates in a context
+
+The *Map templates* extension is enabled by the admin in the [Application Context](application-context.md) wizard. In particular, this happens in the [third step](application-context.md#configure-plugins) of the wizard and after the extension has been added to the *Enabled Plugins* list.
+
+<img src="../img/map-templates/configure-templates.jpg" class="ms-docimage" style="max-width:500px;"/>
+
+As soon as the **Configure templates** button <img src="../img/button/configure-templates-button.jpg" class="ms-docbutton" /> is selected the *Configure templates* window opens allows the admin to manage the templates.
+
+<img src="../img/map-templates/map-templates-window.jpg" class="ms-docimage" style="max-width:500px;"/>
+
+Here the admin can select the templates to moving from the *Available Templates* list to the *Enabled Templates* list through the central bar, as follows:
+
+<img src="../img/map-templates/moving-templates.jpg" class="ms-docimage" style="max-width:500px;"/>
+
+### Uploading the template
+
+In order to upload a new template the admin can select the **Upload new template** button <img src="../img/button/upload-button.jpg" class="ms-docbutton"/> and the **Upload new template** window appears:
+
+<img src="../img/map-templates/upload-map-template.jpg" class="ms-docimage" style="max-width:300px;"/>
+
+Here the admin, in order to import the file, can drag and drop it inside the import screen or select it from the folders of the local machine selecting the import screen.
+
+<img src="../img/map-templates/import-screen.jpg" class="ms-docimage" style="max-width:300px;"/>
+
+!!! warning
+    
+    The file that the admin can download are:
+
+    * The MapStore JSON in `json` format
+
+    * The WMC (Web Map Context) file in `xml` format
+
+The admin can also add **Thumbnail**, **Name**, **Description** and **Groups permissions** as is describe [here](resources-properties.md)
+
+### Customize the template
+
+The admin can also delete or modify the template through the buttons on the left of each templates in the *Configure templates* window.
```

Comment:
The admin can also delete or modify an existing template through the buttons that are available on the left side of each templates item inside the *Configure Templates* UI.